tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -46,7 +46,7 @@ def create_dag(database_name, table_args, dataset):
     dag_name = f"load_file_{dataset_name}_into_{database_name}"
 
     with DAG(dag_name, schedule_interval=None, start_date=START_DATE) as dag:
-        chunk_size = int(os.getenv("ASTRO_CHUNK_SIZE", DEFAULT_CHUNK_SIZE))
+        chunk_size = int(os.getenv("ASTRO_CHUNK_SIZE", str(DEFAULT_CHUNK_SIZE)))
```

Comment:
```suggestion
        chunk_size = int(os.getenv("AIRFLOW__ASTRO_SDK__CHUNK_SIZE", str(DEFAULT_CHUNK_SIZE)))
```

Reply:
@utkarsharma2 this isn't really an astro config. It's just an env variable we set for the benchmarking.